plugins/smileystats.py

```diff
@@ -49,7 +57,17 @@ def smileyCount(self, ctx, msg):
             
             ctx.msg("{:s} has used a total of {:d} smileys.".format(speaker, amt), msg.replyTo)
             return
-        if num == 2 and bodyParts[0].lower() == "all" and bodyParts[1].lower() == "everyone":
+        if num == 2 and smiley.lower() == "all" and speaker.lower() != "everyone":
+            query = self.db.execute("SELECT SUM(count) FROM SmileyCount WHERE speaker = ?", (speaker.lower(),))
+            res = query.fetchone()
+            if not res:
+                amt = 0
+            else:
+                amt = res[0]
+            
+            ctx.msg("{:s} has used a total of {:d} smileys.".format(speaker, amt), msg.replyTo)
+            return
+            if num == 2 and smiley.lower() == "all" and speaker.lower() == "everyone":
```

Comment:
Unless GitHub is showing the indentation incorrectly, this is indented one too far.